snipe-it/start.sh

```diff
@@ -12,4 +12,21 @@ if [ -n "${MYSQL_PASSWORD_FILE}" ]; then
   export MYSQL_PASSWORD=$(cat $MYSQL_PASSWORD_FILE)
 fi
 
+# If the Oauth DB files are not present copy the vendor files over to the db migrations
+if [ ! -f "/var/www/html/database/migrations/*create_oauth*" ]
+then
+  cp -ax /var/www/html/vendor/laravel/passport/database/migrations/* /var/www/html/database/migrations/
```

Comment:
Why do we need this?

Reply:
This is the code from /startup.sh which for now is not running the migrations as its below the exec code.

and since we are running migration before the startup.sh script we need to make sure the passport related migrations are present before running the migrations

Reply:
the code `exec supervisord -c /supervisord.conf` in startup.sh is stopping the execution of the `php artisan migrate` commands. so it has been added here so that the table for oauth gets migrated.

Reply:
In branch 5.1.5 of the original snipe-it repo, this logic is called _before_ the `exec supervisord...` command.

Reply:
Strictly speaking, the Dockerfile. in the main branch of this repo points to the snipe-it 5.1.4 image. But in snipe-it 5.1.4, same thing as in 5.1.5.  I've checked in the develop branch, and it hasn't changed.
So this line should not be necessary.

Reply:
https://github.com/snipe/snipe-it/blob/v5.1.4/docker/startup.sh 

in `/startup.sh`
the command to copy the migration files is before the `exec supervisord -c /supervisord.conf` line but the command to migrate `php artisan migrate --force` the migrations is after above script. so the migration command is not executed by `/startup.sh`. 

in `/start.sh`
Since we are now running the migrate command in `start.sh` we just need to make sure the migrations file for oauth are copied before we run the `php artisan migrate --force` command.